backend/src/models.py

```diff
@@ -0,0 +1,12 @@
+# Create your models here.
+from django.db import models
+import uuid
+
+class Rule(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    name = models.CharField(max_length=50, null=True, blank=True)
+    frequency = models.CharField(max_length=50, null=True, blank=True)
+    amount = models.CharField(max_length=50, null=True, blank=True)
```

Comment:
Double-check the story, there are more than just these 4 fields (version, userid, I think 1 more, again check the story)

Reply:
We'll get this next time, presumably with the create API.